metadata.rb

```diff
@@ -13,8 +13,8 @@
 supports 'scientific'
 supports 'oracle'
 
-depends  'poise-python', '>= 1.5'
+depends  'pyenv', '~> 3.1'
```

Comment:
This makes me a bit nervous. We should probably make this >= 3.1. Otherwise this may conflict with another cookbook and prevent systems from being able to depsolve